kit/aks/buildingblocks/aks-starterkit-composition/starterkit/main.tf

```diff
@@ -94,12 +141,21 @@ resource "time_sleep" "wait_45_seconds" {
   create_duration = "45s"
 }
 
+# Fetch the GitHub building block after creation to get outputs
+data "meshstack_building_block_v2" "repo_data" {
+  depends_on = [time_sleep.wait_45_seconds]
```

Comment:
d: we still have to implement in the tf provider to poll until successful provisioning of a new BB, right? that would eliminate this sleep abomination...

similar to https://github.com/meshcloud/terraform-provider-meshstack/issues/34